### ISSUE TITLE
Is this repository/project active/dead?

### ISSUE THREAD

**Issue by matanshukry** [orientechnologies/spring-data-orientdb#87]
As far as I can tell the last commit in this repo has been around 3 years ago.

Version 3 of orientdb is already out; can I assume this repository will not work with it? what about spring version?

There seems to even be a PR to update the repo to 2.2.x, which no one touched for 2 years now!

If not - what is the recommended approach to using Orientdb in java with spring? Is there an example project?

**Comment by adepase** [comment]
You can change the dropdown on the top left named branch and choose 3.0.x to see latest 3.0.x commits.
Nevertheless, I cannot find a Maven repository to download 3.0.x modules (I need autoconfigure, above all) from.

**Comment by matanshukry** [comment]
It seems the 3.0.x modules are available, just versioned differently (0.13/0.14 latest)
https://mvnrepository.com/artifact/com.orientechnologies/spring-data-orientdb-graph

That too seems to be outdated though, being released 2 years ago.

Personally I created my own spring-orientdb-data module and using it; Couldn't find any other solution.

**Comment by adepase** [comment]
No, 0.14 was already a version for 2.x, I'm using it since a couple of years, at least.
The newest versions, according with the recent commits in github are currently versioned as 0.14-3.0.15-SNAPSHOT .
They are not yet aligned everywhere, because in the submodules that is the version, but in the parent one the version is 0.14-3.0.18-SNAPSHOT.
I agree, versions on Maven repositories don't seem to be updated to 3.0.X, it's a pity.

**Comment by gunaskr** [comment]
We are trying to use branch 3.0.x. The version is an issue,  as modules have 0.14-3.0.15-SNAPSHOT , whereas the parent pom has 0.14-3.0.18-SNAPSHOT.
It is also not compatible with spring-data-commons 2.1.5.RELEASE, which we are using. 
I fixed these two issues and its working fine, at least the basic stuff like named queries and such. 
I would be happy to raise a PR if you guys are interested.
**And if possible let us know what other issues we should expect when using 3.0.x branch in production.**

**Comment by financelurker** [comment]
Well, seems pretty dead to me.

**Comment by lvca** [comment]
Hi guys, the entire OrientDB project is pretty much dead (look at the contributions and the people involved in them). I suggest you switch to ArcadeDB (https://github.com/ArcadeData/arcadedb), born initially as a fork of OrientDB, but then evolved into something much more powerful. There are some ex-orientdb people involved with it, it's still Apache2, the same SQL, and most of the OrientDB features are right there. 

Forgot to mention, this is Luca, the original OrientDB founder, now working with ArcadeDB.

If not ArcadeDB, I'd suggest using something else with good support in terms of community.

**Comment by financelurker** [comment]
Thx for the information!

Is ArcadeDB a fork of the former OrientDB community edition or enterprise edition?
I'd be interested in graph DB functionality with replication/sharding/scaling and also access-control functionality...

**Comment by lvca** [comment]
ArcadeDB supports replication/scaling (https://docs.arcadedb.com/#High-Availability) but not sharding. Funny thing, sharding with OrientDB was never 100% completed :-) Access Control is there (https://docs.arcadedb.com/#Security)

**Comment by adepase** [comment]
But this is a partial fork, as far as I understand and there are important
differences.
For example, there are no stored procedures (server side scripting).
In my case I have an important server side scripting part.
Tell me that I'm wrong, please or that there will be a comparable (and
possibly compatible) server side scripting on ArcadeDB.

And is there the possibility to import data from OrientDB exports or
planning for porting from OrientDB data?

Moreover: is it really dead or just changing?
I see there is a really new OrientDB 3.2.11 GA Community Edition (October
19th, 2022), and I read that the features in the Enterprise version should
be ported to the open source version (
https://orientdb.org/enterprise-agent-open-source).
Is that correct?
It seems it will become an open source only project, probably missing
enterprise support.
Which is not good, but I cannot see other ready alternatives to replace it
in the short time


Il giorno gio 27 ott 2022 alle ore 18:31 Luca Garulli <
***@***.***> ha scritto:
